### DISCUSSION TITLE
BDFR just hangs

### DISCUSSION THREAD

**Discussion by Joeweav** [Serene-Arc/bulk-downloader-for-reddit#623] [Q&A]
I am not sure if this is a bug or something to chat about.  But every night when BDFR runs, it hangs on at least one item.  I keep logs and exclude files, so when this happens I obviously add the thing it choked on to the exclude file, but it will just choke on something else the next night.  I cranked the verbosity up to -v -v and I get this:

[2022-03-29 04:27:50,630 - bdfr.downloader - DEBUG] - Attempting to download submission xyzabc

And that is all she wrote.  The only option is to control c to break out of that run, and on rare occasion that does not work and I have to close the window.

I looked at the --max-wait-time option but the default is reasonable and I have let this go for 10 hours and it has remained stuck so I don't think that is a solution.

Any thoughts for a solution?  I may try adding a couple more -v's to the mix in the meantime.

**Comment by Serene-Arc** [comment]
I'll need to know what the submission ID is to investigate this. It could be any number of things, this really isn't enough information
